src/View/Helper/FormHelper.php

```diff
@@ -1044,6 +1047,19 @@ public function control(string $fieldName, array $options = []): string
         $options = $this->_parseOptions($fieldName, $options);
         $options += ['id' => $this->_domId($fieldName)];
 
+        if (
+            !empty($options['label']) && $options['type'] != 'hidden' ||
+            !empty($options['placeholder']) && $options['type'] != 'hidden'
+        ) {
```

Comment:
These empty() calls could eat `'0'` which *could* be a label or placeholder. Doing a comparison on the input type should be enough as an empty aria-label won't hurt.

```suggestion
        if ($options['type'] != 'hidden') {
```